public/js/room.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function() {
+    function getRooms() {
```

Comment:
Again, probably no reason to define getRooms as a function unless you are going to be using it more than once.